mongodb/src/main/kotlin/com/example/mongodb/messageRepository/MessageRepository.kt

```diff
@@ -0,0 +1,28 @@
+package com.example.mongodb.messageRepository
+
+import com.example.mongodb.model.Message
+import com.example.mongodb.model.TimeTracker
+import mu.KLogging
+import org.springframework.data.mongodb.repository.MongoRepository
+import org.springframework.stereotype.Repository
+import java.time.Instant
+
+interface MessageRepositoryInterface : MongoRepository<Message, String> {}
```

Comment:
Any reason you cant just do `class MessageRepository(..) : MongoRepository<Message, String>` ?